### PR TITLE
Refactor command and service

### PR DESCRIPTION
**Pull request by kenoss**
## Check list

- [x] I wrote [CHANGELOG.md](./CHANGELOG.md) if the pull request adds/modifies features.

Internal refactoring only.

## Summary

Refactor command and service. Added service server for test.

### CODE REVIEW

**Comment by kenoss** [comment]
@notogawa Could you review it, please.

File: pyproject.toml [hunk 1]
```diff
@@ -32,10 +32,13 @@ python = "^3.6"
...
 # For example, the version constraint of python package `Pillow` should be `^5`.
 Pillow = "^5"
+dataclasses = {version = "^0.8", python = "< 3.7"}
 
 [tool.poetry.dev-dependencies]
 Sphinx = "^3.5.3"
-pyproject-indirect-import-detector = "^0.1.1"
+cryptography = "^3.4.7"
+pyOpenSSL = "^20.0.1"
+pyproject-indirect-import-detector = "^0.1.2"
```
Comment (kenoss) [inline pyproject.toml:41]: Because we need https://github.com/kenoss/pyproject-indirect-import-detector/pull/18 .

**Review by notogawa — APPROVED** [review]
LGTM

**Comment by kenoss** [comment]
Thanks. I'll merge it.

**Pull request merged**